### ISSUE TITLE
[Enhancement] Root user access in App Container

### ISSUE THREAD

**Issue by k-304** [sysu-ipv6/UNIT3D-Community-Edition#43]
In the Dockerfile you switch to the user www-data as 2nd last step. 
If you want to access the container later for example to tweak some individual code, you do not have access to edit any file in /app because they belong to the root user but you are user "www-data".

It would be nice to have root console access to edit files within /app of the "app" container.

**Comment by howardlau1999** [comment]
The docker image is meant to be used in production so we stripped almost everything unnecessary off. Also, it's not recommended that you modify code directly in the container because every restart resets the code. Instead, you can copy the code directory out from the container first then mount that folder back into container. So you can modify your code outside the container and make them survive restarts.

For example, you may first `docker ps` to get the id of the running UNIT3D container. Then you can copy the files out using `docker cp $CONTAINER_ID:/app /path/to/your/host`. Fix the owner or permissions as needed. Then you can kill the original docker image and re-run it using `docker run -it -d -p 8080:1215 -v /path/to/your/host:/app sysuipv6/syspt`.

P.S.: We modify the code a lot so chances are that you will have trouble updating code from the upstream repository. But you are free to use the Dockerfile to build your own docker image.

To sum up:

1. Do not modify code in the container.
2. Copy the code out to your host and mount them back in container.

**Comment by howardlau1999** [comment]
By the way, the `USER www-data` can be overwritten using `docker run/exec --user=root`

**Comment by k-304** [comment]
Thanks for the tips, i'm pretty new to docker and still learning. This will help me a lot.
